lib/HLSL/DxilCondenseResources.cpp

```diff
@@ -120,7 +120,7 @@ class DxilResourceRegisterAllocator {
 
   template <typename T>
   static bool
-  AllocateRegisters(const std::vector<std::unique_ptr<T>> &resourceList,
+  AllocateRegisters(DxilModule &DM, const std::vector<std::unique_ptr<T>> &resourceList,
     LLVMContext &Ctx, SpacesAllocator<unsigned, T> &ReservedRegisters,
```

Comment:
Ctx is unused now, besides, if needed, can be retrieved from the module, since you've added that.